bin/build-plugin-zip.sh

```diff
@@ -26,7 +26,7 @@ warning () {
 	echo -e "\n${YELLOW_BOLD}$1${COLOR_RESET}\n"
 }
 
-status "💃 Time to release Gutenberg 🕺"
+status "💃 Time to build Gutenberg 🕺"
```

Comment:
I guess it would be fine to make it even more descriptive like:
> Time to build Gutenberg plugin ZIP file

Given that this script should be used mostly through release automation, we might consider renaming from `npm run package-plugin` to something like `npm run build:plugin-zip`.

Reply:
Makes sense, updated in https://github.com/WordPress/gutenberg/pull/23063/commits/1a71c531114d010eb5dc5a6d35b1d9ddf61f19e1.